app/helpers/user-really-may.js

```diff
@@ -0,0 +1,10 @@
+import Helper from '@ember/component/helper';
+import { inject as service } from '@ember/service';
+
+export default class UserReallyMay extends Helper {
```

Comment:
Provide a named argument `checkImpersonator` on the `user-may` helper instead of introducing a separate helper?